src/App.js

```diff
@@ -12,13 +12,79 @@ const goodsFromServer = [
   'Honey',
   'Jam',
   'Garlic',
+  'Milk',
+  'Bear',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: ['Jam'],
+  }
+
+  addToSelected = (product) => {
+    if (!this.state.selectedGoods.includes(product)) {
+      this.setState(state => ({
+        selectedGoods: [...state.selectedGoods, product],
+      }));
+    } else {
+      this.setState(state => ({
+        selectedGoods: state.selectedGoods.filter(good => good !== product),
+      }));
+    }
+  }
+
+  clearSelectedGoods = () => {
+    this.setState({ selectedGoods: [] });
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <h1 className="SelectedGoods">
+          Selected goods:
```

Comment:
You forgot to add "No goods selected" from 3 task
3. Add h1 initially saying No goods selected

Reply:
add `no goods selected`